packages/cli/package.json

```
@@ -2,6 +2,10 @@
   "name": "@quality-metrics/cli",
   "version": "0.0.1",
   "dependencies": {
-    "bundle-require": "^4.0.1"
+    "bundle-require": "^4.0.1",
+    "chalk": "^5.3.0",
+    "yargs": "^17.7.2",
+    "zod": "^3.22.1",
+    "@quality-metrics/models": ">=0.0.1"
```

I guess depending on major versions is safer, in case we make breaking changes.

```suggestion
    "@quality-metrics/models": "^0.0.1"
```

---

Same applies to `package.json`s in other projects.